FreeRTOS/Test/CMock/smp/single_priority_timeslice/single_priority_timeslice_utest.c

```diff
@@ -561,26 +551,11 @@ void test_task_create_tasks_equal_priority( void )
     /* Create a new task of equal priority */
     xTaskCreate( vSmpTestTask, "SMP Task", configMINIMAL_STACK_SIZE, NULL, 1, &xTaskHandles[configNUMBER_OF_CORES] );
 
-    /* The new task will begin in the ready state */
-    int32_t core = -1;
-
     /* Verify the last created task runs on each core or enters the ready state */
     for (i = 0; i < configNUMBER_OF_CORES; i++) {
         
-        xTaskIncrementTick();
-
-        verifySmpTask( &xTaskHandles[configNUMBER_OF_CORES], (core == -1) ? eReady : eRunning, core );
-
-        /* Track wrap around to the ready state */
-        if ((i % configNUMBER_OF_CORES) == (configNUMBER_OF_CORES - 1)) {
-            core = -1;
-        } else {
-            core = (i % configNUMBER_OF_CORES) + 1;
-        }
-
-        if ((i % configNUMBER_OF_CORES) == 0) {
-            core = 1;
-        }
+        xTaskIncrementTick_helper();
+        verifySmpTask( &xTaskHandles[configNUMBER_OF_CORES], eRunning, i );
```

Comment:
I think the test case description above needs to updated to match the code change